### DISCUSSION TITLE
To reflect existing table relations in Directus.

### DISCUSSION THREAD

**Discussion by azoom-keishi-ichida** [directus/directus#22103] [Feature Requests]
### Summary

I am considering using Directus to manage the database. However, currently I am using PostgreSQL for the existing database. I want to manage that database with Directus, but I couldn't manage the complex data with it effectively. To manage it via the admin UI of Directus, how should I proceed? Do I need to make settings that indicate the existence of relations for Directus_fields or Directus_relations databases?

Create data with relations in the existing database.
Connect to the existing data using Directus.
Confirm relations from Directus's admin UI.
The data is recognized as simple tables and I am unable to establish relationships.

### Motivation

I am considering using Directus to manage the database. However, currently I am using PostgreSQL for the existing database. I want to manage that database with Directus, but I couldn't manage the complex data with it effectively. To manage it via the admin UI of Directus, how should I proceed? Do I need to make settings that indicate the existence of relations for Directus_fields or Directus_relations databases?

Create data with relations in the existing database.
Connect to the existing data using Directus.
Confirm relations from Directus's admin UI.
The data is recognized as simple tables and I am unable to establish relationships.

### Detailed Design

I am considering using Directus to manage the database. However, currently I am using PostgreSQL for the existing database. I want to manage that database with Directus, but I couldn't manage the complex data with it effectively. To manage it via the admin UI of Directus, how should I proceed? Do I need to make settings that indicate the existence of relations for Directus_fields or Directus_relations databases?

Create data with relations in the existing database.
Connect to the existing data using Directus.
Confirm relations from Directus's admin UI.
The data is recognized as simple tables and I am unable to establish relationships.

### Requirements List

I am considering using Directus to manage the database. However, currently I am using PostgreSQL for the existing database. I want to manage that database with Directus, but I couldn't manage the complex data with it effectively. To manage it via the admin UI of Directus, how should I proceed? Do I need to make settings that indicate the existence of relations for Directus_fields or Directus_relations databases?

Create data with relations in the existing database.
Connect to the existing data using Directus.
Confirm relations from Directus's admin UI.
The data is recognized as simple tables and I am unable to establish relationships.

### Drawbacks

While Directus allows you to incorporate existing databases and utilize their relationships, it does not offer direct management or modification of the underlying database structure or data. Therefore, it would be more accurate to say that Directus enables you to leverage existing database relationships rather than directly manage the databases themselves.

### Alternatives

Directus allows direct manipulation of its directus_relations and directus_fields tables, enabling users to customize and manage the relationships and fields within the Directus interface.

### Adoption Strategy

Directus allows direct manipulation of its directus_relations and directus_fields tables, enabling users to customize and manage the relationships and fields within the Directus interface.

**Comment by br41nslug** [comment]
Closing this as it does not appear to be a feature request. For questions feel free to ask in the community support discord https://directus.chat/
